src/utils/public/index.ts

```diff
@@ -1,5 +1,7 @@
-export * from "./percentage";
-export * from "./tick";
+export { TokenUSDPrices } from "../token-price";
```

Comment:
wasn't sure what the semantics were behind putting things in public or not?

Reply:
i think @odcheung will do a pass on stuff like this when he refactors the SDK